src/App.tsx

```diff
@@ -31,6 +31,7 @@ type Props = {
 	baseUrl: string;
 	pillar: Theme;
 	isClosedForComments: boolean;
+	isFullWidth?: boolean;
```

Comment:
where does this prop come from?

Reply:
It would be passed in from DCR or any other consuming frontend, for example we'd add the prop on DCR [here](https://github.com/guardian/dotcom-rendering/blob/b3211ac220a7a7f82dff7d44a4f1b5813127d19b/src/web/components/Discussion.tsx)

Reply:
would we ever want `discussion-rendering` to not be full width?

Reply:
I wasn't 100% sure on this, but knew it was being used other frontend services as well, and wasn't sure how their implementation & containers would work or if they would place nice with it. The prop is mostly just a safety guard for that reason, but it could be removed and leave it up to someone checking when they update